src/app/components/ActiveUserLoanContainer/components/ActiveLoanTableContainer/index.tsx

```diff
@@ -146,9 +147,13 @@ export function ActiveLoanTableContainer(props: Props) {
     }
   }, [expandedId]);
 
+  useEffect(() => {
+    window.addEventListener('resize', () => setWindowSize(window.innerWidth));
+  }, []);
+
```

Comment:
Listener is not unsubscriber which opens issue with memory leak.
Leaving and opening page multiple times will add multiple resize listeners to page and gradually make it impossible for user to browser.
Example on how to unsubscribe:
```
  useEffect(() => {
     const listener =  () => setWindowSize(window.innerWidth);
     window.addEventListener('resize',listener);
     return () => window.removeEventListener('resize', listener);
  }, []);

```

But in our case block should be removed entirely, because it's better use css media breakpoints to hide elements instead of watching javascript.

```suggestion

```